desktop-exporter/app/routes/trace-view.tsx

```diff
@@ -111,4 +80,59 @@ export default function TraceView() {
       </GridItem>
     </Grid>
   );
+}
+
+function orderSpans(spanTree: TreeItem[]): SpanWithUIData[] {
+  let orderedSpans: SpanWithUIData[] = [];
+
+  for (let root of spanTree) {
+    let stack = [
+      {
+        treeItem: root,
+        depth: 0,
+      },
+    ];
```

Comment:
We are sorting each set of children, but not the set of root nodes we are starting with. We might wish to sort `spanTree` before we enter this loop so that the root span is first and all missing spans come after.

Reply:
The array-to-tree implementation takes care of that, as any spans left in the missing set are added to the rootSpans array only after all the spans have been processed, and the for loop exits. I added a comment both in `trace-view` and `array-to-tree`